rs/tests/ckbtc/Cargo.toml

```diff
@@ -7,13 +7,17 @@ description.workspace = true
 documentation.workspace = true
 
 [dependencies]
```

Comment:
Here for a lack of a better place: Does-it make sense to even have a `Cargo.toml` file for system tests? AFAIK those targets can only be run using bazel (and the appropriate `config` flag) and Cargo compatibility is not required here. For example, we don't have any Cargo file in `rs/tests/cross_chain`.

Reply:
I don't know. But for consistency, the binaries were declared in rs/tests/Cargo.toml, and now moved here to avoid dependency loop.

Also, I use cargo with my editor (vim), so it is beneficial to be able to show errors while editing.

Reply:
Ok, if that helps you, I'm happy to keep it!